drivers/display/Kconfig.st7789v

```diff
@@ -0,0 +1,68 @@
+# Kconfig - ST7789V display driver configuration options
+
+#
+# Copyright (c) 2019 Marc Reilly <marc@cpdesign.com.au>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ST7789V
+	bool "ST7789V display driver"
+	depends on SPI
+	help
+	Enable driver for ST7789V display driver.
```

Comment:
Please indent the help text by two spaces

Reply:
done